barretenberg/cpp/src/barretenberg/vm/avm/generated/circuit_builder.cpp

```diff
@@ -30,15 +49,48 @@ AvmCircuitBuilder::ProverPolynomials AvmCircuitBuilder::compute_polynomials() co
                                               /*make shiftable with offset*/ 1 };
                        }
                    }));
+
     // catch-all with fully formed polynomials
     AVM_TRACK_TIME(
         "circuit_builder/init_polys_unshifted", ({
             auto unshifted = polys.get_unshifted();
-            bb::parallel_for(unshifted.size(), [&](size_t i) {
+
+            // An array which stores for each column of the trace the smallest size of the
+            // truncated column containing all non-zero elements.
+            // It is used to allocate the polynomials without memory overhead for the tail of zeros.
+            std::array<size_t, Row::SIZE> col_nonzero_size{};
```

Comment:
You don't need the {} for size_t I think. But let's leave it just in case.

Reply:
Actually you need them otherwise it will be uninitialized like a C array.